Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -261,6 +261,11 @@ void VideoConfig::VerifyValidity()
       iStereoMode = 0;
     }
   }
+  if (!backend_info.bSupportsBBox && g_ActiveConfig.bBBoxEnable)
+  {
+    PanicAlertT("Your graphics card does not support bounding box emulation this game requires. "
+                "You will experience bugs and crashes. This is not a Dolphin bug.");
```

Comment:
You have a point about "bugs or freezes", though. Some of the affected games might not have both, so "bugs and freezes" isn't true for every game. But on the other hand, you can see it as the combination of all those games having "bugs and freezes", which is true... I suppose either wording is fine.